test/compflags/diten/savec.chpl

```diff
@@ -1,26 +1,30 @@
 const outdir = "savec_output";
 const filename = "savec.chpl";
 
-var ret: int;
+proc mysystem(cmd: string): int {
+  use Spawn;
+  var sub = spawnshell(cmd);
```

Comment:
I generally prefer `spawn()`, but since no user input is involved here, it doesn't really matter.